ocaml/file_formats/cms_format.ml

```diff
@@ -26,8 +26,10 @@ type cms_infos = {
   cms_comments : (string * Location.t) list;
   cms_sourcefile : string option;
   cms_builddir : string;
+  cms_loadpath : Load_path.paths;
```

Comment:
As discussed, I'm not sure we need to store this. The build system can likely provide it. Please add a CR saying we might be able to get rid of this later.

Reply:
Agreed, CR added.